### ISSUE TITLE
Make test-cargo CI run on Windows

### ISSUE THREAD

**Issue by sffc** [unicode-org/icu4x#3109]
When run on Windows, I get errors such as

```
cp -p "D:\a\icu4x\icu4x\docs\tutorials\cargo_tests/target"/release/demo_buffer bin/demo_buffer
./bin/icu4x-datagen \
	--format blob \
	--keys-for-bin bin/demo_buffer \
	--locales my en-ZA \
	--cldr-tag latest \
	--overwrite \
	--out buffer_data.postcard
Error: bin/demo_buffer

Caused by:
    The system cannot find the file specified. (os error 2)
```

Need to debug. Probably easy to do locally; just get `cargo make test-cargo-tutorial-local` to succeed.

**Comment by hiralkhatik123** [comment]
may be The system cannot find the file specified. the file path is incorrect. Verify the file's path.

**Comment by mihnita** [comment]
There is no `cp` command in the normal Windows shell.
There is a `cp` command in PowerShell, cygwin, WSL, or some kind of Linux-like shell. Not very standard.

The error does not look like a PowerShell one.

Is there a link to some full log?

My suspicion is the double quotes on the source.

They should either wrap the whole path, or not have them at all

No:
```
cp -p "D:\a\icu4x\icu4x\docs\tutorials\cargo_tests/target"/release/demo_buffer bin/demo_buffer
```
Yes:
```
cp -p "D:\a\icu4x\icu4x\docs\tutorials\cargo_tests/target/release/demo_buffer" bin/demo_buffer
```
Or:
```
cp -p D:\a\icu4x\icu4x\docs\tutorials\cargo_tests/target/release/demo_buffer bin/demo_buffer
```

**Comment by swar09** [comment]
Is this issue open ? I would like to work in this . @sffc

**Comment by sffc** [comment]
@swar09 Yes, this issue is still open. Do you have a Windows system to test this on?

Please make `cargo make test-cargo` work on Windows. As a bonus, please also make the CI job `test-cargo` work on Windows.

**Comment by swar09** [comment]
Yes I have windows system, sure I will update soon.

**Comment by swar09** [comment]
@sffc  
I attempted to reproduce the issue on my Windows system. My approach is: 

  1.Update the Makefile with cross-platform compatible syntax that works on both Unix-like systems and Windows
  2. Verify the changes work locally on Windows
  3. Update the CI job to ensure it passes on all platforms

**Comment by sffc** [comment]
Thank you. We should make all of the examples in `test-cargo` be cross-platform instead of having platform-specific makefiles. The main things that probably need to remain platform-specific are the LLVM LTO tests like js-tiny, which I think are not reachable by `test-cargo`.

**Comment by swar09** [comment]
Few
[cargo-make] INFO - Build Done in 33.83 seconds. 

`cargo make test-cargo` works on Windows now , now i will attempt CI job .  I will create PR soon.
